doc/source/serve/advanced.rst

```diff
@@ -421,3 +421,24 @@ in :mod:`serve.start <ray.serve.start>`:
    Using the "EveryNode" option, you can point a cloud load balancer to the
    instance group of Ray cluster to achieve high availability of Serve's HTTP
    proxies.
+
+Routing
+=======
+
+Ray Serve supports capturing path parameters.  For example, in a call of the form
+
+.. code-block:: python
+
+    client.create_endpoint("my_endpoint", backend="my_backend", route="/api/{username}")
+
+the ``username`` parameter will be accessible in your backend code as follows:
+
+.. code-block:: python
+
+    def my_backend(request):
+        username = request.path_params["username"]
+        ...
+
+Ray Serve uses Starlette's Router class under the hood for routing, so type
+conversion for path parameters is also supported.  For more details, see Starlette's
```

Comment:
I think it's worth having a "more advanced" example that uses multiple path parameters and also does the typing. Probably as a second example so the first is very readable.